spec/views/catalog/_home_text.html.erb_spec.rb

```diff
@@ -17,6 +17,10 @@
     it "has a links to bulk ingest resources" do
       expect(rendered).to have_link "Bulk Ingest"
     end
+
+    it "has a Media Reserves panel" do
```

Comment:
If you remove the box above, you should remove this test too.

Reply:
thanks @tpendragon I removed it.